b64d/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "b64d"
+version = "0.1.0"
+edition = "2021"
+authors = ["Logan Wemyss"]
+description = "Decode a base64 encoded file"
+repository = "https://github.com/knassar702/hacks"
+license = "GPL-3.0-or-later"
+keywords = ["base64"]
+categories = ["command-line-utilities"]
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[profile.release]
```

Comment:
Is this all really necessary? Especially abort on panic... really? space can't be that big of an issue

Reply:
agree, but we don't want a big size difference between go/Rust version
Go : 1.9M
Rust: 7.6M

we can disable remove abort line